single_level.py

```diff
@@ -27,14 +27,59 @@ def get_note_types(resource):
 		else:
 			note_types.append(note["jsonmodel_type"])
 	return note_types
+	
+def get_extent_number(resource):
+	extent_number = []
+	for extent in resource["extents"]:
+		if extent["number"] > 0:
+			extent_number.append("true")
+		else:
+			extent_number.append("false")
+	return extent_number
+	
```

Comment:
Can you explain your reasoning behind looking for an extent number rather than a container summary? I'm not suggesting you're wrong, just wondering what the thinking was!